src/test/resources/com/arpnetworking/metrics/mad/parsers/QueryLogParserV2gTest/testEmpty.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": "2g",
+    "dimensions": {},
```

Comment:
Hrm. So in the code this is nullable but in the _empty.json_ it's set. I prefer it this way, but then you should definitely change the code to require this field. Is there a reason to support it as null/missing?